pages/common/xargs.md

```diff
@@ -10,6 +10,10 @@
 
 `{{arguments_null_terminated}} | xargs -0 {{command}}`
 
-- Example: list unneeded packages with deborphan and remove them with apt-get:
+- delete all files that start with 'M'
 
-`sudo deborphan | xargs sudo apt-get remove`
+`ls M* | xargs rm`
+
+- assign incoming arguments to a variable so that they may be used in the middle of a command
```

Comment:
I don't think the explanation is too generic. My problem with it is that it doesn't make the syntax clear, e.g. what each of the two % mean in the command. If that was clear, the fact that it doesn't describe the `cp` command is fine IMO, since when we need auxiliary commands like `ls` or `echo` for piping demonstrations, we don't describe what they do either.